src/js/common/schemaform/widgets/GenderWidget.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
I don't think we need a widget for gender. Should be able to work the same as the relationship field.